README.md

```diff
@@ -32,8 +32,9 @@ Last stable versions of packages that are production-ready. These libraries prov
 ## Need help
 
 - For reference documentation visit the [Azure SDK for C++ documentation](https://azure.github.io/azure-sdk-for-cpp).
-- For tutorials, samples, quick starts and other documentation, visit [Azure for C++ Developers](https://docs.microsoft.com/azure/).
-- For build reports on code quality, test coverage, etc, visit [Azure SDK for C++]<!--https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/index.html-->.
+
+- For tutorials, samples, quick starts and other documentation, visit [Azure for Developers](https://docs.microsoft.com/azure/).

```

Comment:
I know we will likely change the link later, but keep the `C++` in the link text.
```suggestion
- For tutorials, samples, quick starts and other documentation, visit [Azure for C++ Developers](https://docs.microsoft.com/azure/).
```

Reply:
lol, I removed the C++ because it is missleading.. It takes you to some place where there are no C++